projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6548,9 +6548,9 @@ public static function is_rewind_enabled() {
 		if ( ! static::connection()->has_connected_owner() ) {
 			return false;
 		}
-
 		$rewind_enabled = get_transient( 'jetpack_rewind_enabled' );
-		if ( false === $rewind_enabled ) {
+		$recheck        = ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) && 0 === $rewind_enabled;
```

Comment:
Looks like this isn't quite right.
```suggestion
		$recheck        = ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) && '0' === $rewind_enabled;
```